app/models/message.rb

```diff
@@ -85,6 +88,20 @@ def self.create_from_queue(body, import = false)
 
     message.set_state!
   end
+
+  def retry(message)
+    if message.retried_at.blank?
```

Comment:
use guard clause don't wrap in an if block;

` return if message.retried_at.present?`